.rubocop.yml

```diff
@@ -16,6 +16,7 @@ Layout/IndentHash:
 Metrics/LineLength:
   Exclude:
     - "test/**/*.rb"
+    - "lib/stripe/**/*.rb"
```

Comment:
lol — maybe we should just get rid of this lint instead?

Reply:
I'm hoping to scope things down a bit by moving everything to a `resources` dir, and moving the lint exemption there